regression-test/suites/insert_p2/test_group_commit_insert_into_lineitem_multiple_client.groovy

```diff
@@ -50,7 +50,7 @@ suite("test_group_commit_insert_into_lineitem_multiple_client") {
         }
     }
     def insert_table = "test_insert_into_lineitem_multiple_client"
-    def batch = 100;
+    def batch = 90;
```

Comment:
add a NOTICE comment to explain why must set to 90

Reply:
added